packages/story-editor/src/components/library/panes/media/local/mediaRecording/mediaRecording.js

```diff
@@ -54,13 +56,28 @@ function MediaRecording() {
     })
   );
 
+  const enableMediaRecording = useFeature('mediaRecording');
+  const { isTranscodingEnabled } = useFFmpeg();
+
   const onClick = useCallback(() => {
     trackEvent('media_recording_toggled', {
       status: isInRecordingMode ? 'closed' : 'open',
     });
     toggleRecordingMode();
   }, [isInRecordingMode, toggleRecordingMode]);
 
+  if (!enableMediaRecording) {
```

Comment:
```suggestion
  if (!enableMediaRecording || !isTranscodingEnabled) {
```

Reply:
This check is specifically done a few lines down in order to a) make removing the feature flag code in the future easier b) provide a contextual explanatory inline comment before the check.

Reply:
We don't do this in other places. See
https://github.com/GoogleForCreators/web-stories-wp/blob/c7450e9d5da803f970a1931ab83b273dcc908b07/packages/story-editor/src/components/panels/design/captions/captions.js#L141
https://github.com/GoogleForCreators/web-stories-wp/blob/c7450e9d5da803f970a1931ab83b273dcc908b07/packages/story-editor/src/components/panels/design/pageBackgroundAudio/backgroundAudio.js#L64

Reply:
Which is not nice but also not the end of the world